resources/extruders/teraform_x_120.def.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "X 120",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "teraform_inventor",
+        "position": "0"
+    },
+
+    "overrides": {
+        "extruder_nr": { "default_value": 0 },
+        "machine_nozzle_size": { "default_value": 12 },
+        "material_diameter": { "default_value": 12 }
```

Comment:
12 mm diameter? Uh. Since the machine defines that is has materials, I guess you should also add material profiles in fdm_materials with a diameter of 12 mm?

Reply:
Thank you - was wondering where to add the material... We have submitted a pull request to fdm_materials